src/api/cars/cars.service.ts

```diff
@@ -4,30 +4,99 @@ import { Cars } from './cars.types'
 
 const prisma = new PrismaClient()
 
-export const getAllCars = async () => {
-  const cars = await prisma.cars.findMany({
-    include: {
-      driver: {
-        select: {
-          id: true,
-          first_name: true,
-          last_name: true,
+export async function getAllCars() {
+  try {
+    const cars = await prisma.cars.findMany({
+      include: {
+        driver: {
+          select: {
+            id: true,
+            first_name: true,
+            last_name: true,
+          }
         }
       }
+    });
+    return cars;
+  } catch (error: any) {
+      throw error;
     }
-  })
+}
+
+export async function createCar(input: Cars) {
+  try { 
+    const car = await prisma.cars.create({
+      data: {
+        car_name: input.car_name,
+        type: input.type,
+        driver_id: input.driver_id,
+      }
+    });
+
+    return car;
+
+  } catch(error: any) {
+      throw error;
+    }  
+}
+
+export async function getCarById(id: string) {
+  try {
+    const cars = await prisma.cars.findUnique ({
+      where: {
+        id,
+      },
+    });
+
+    return cars;
+  } catch (error: any) {
+      throw error;
+    }  
+}
+
+export async function getCarByDriverId(driver_id: string) {
+  try {
+    const car = await prisma.cars.findUnique ({
+      where: {
+        driver_id,
+      },
+    });
 
-  return cars
+    return car;
+
+  } catch (error: any) {
+      throw error;
+    }
+}
+
+export async function deleteCar (id: string) {
+  try { 
+    const car = await prisma.cars.delete({
+      where: {
+        id,
+      },
+    });
+
+    return car;
+
+  } catch (error: any){
+      throw error; 
+    }
```

Comment:
balance the bracket according to code format, thx.

Reply:
changed.